site/docs/5_references/1_nifi_cluster/3_node_config.md

```diff
@@ -85,6 +85,8 @@ NodeConfig defines the node configuration
 | image                 | string                                                                                       | Docker image used by the operator to create the node associated. [Nifi docker registry](https://hub.docker.com/r/apache/nifi/)|No|""|
 | imagePullPolicy       | [PullPolicy](https://godoc.org/k8s.io/api/core/v1#PullPolicy)                                | define the pull policy for NiFi cluster docker image.)|No|""|
 | nodeAffinity          | string                                                                                       | operator populates this value if new pvc added later to node [node-affinity](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#node-affinity)|No|nil|
+| seccompProfile        | [SeccompProfile](https://godoc.org/k8s.io/api/core/v1#SeccompProfile)                        | sets the used SeccompProfile | No | [SeccompProfileTypeRuntimeDefault](https://godoc.org/k8s.io/api/core/v1#SeccompProfileType)|
+| securityContext       | [SecurityContext](https://godoc.org/k8s.io/api/core/v1#SecurityContext)                      | sets the used SecurityContext | No | sets AllowPrivilegeEscalation to false and drops all capabilities|
```

Comment:
The documentation should match the description in the CRD:

```go
// seccompProfile overrides the default seccompProfile of the nodes pod
SeccompProfile *corev1.SeccompProfile `json:"seccompProfile,omitempty"`
// securityContext overrides the default container security context for all containers in the pod
SecurityContext *corev1.SecurityContext `json:"securityContext,omitempty"`
```

And the default value of `securityContext` is a description not default value 🤔.